src/SDL/Input/Mouse.hs

```diff
@@ -107,6 +107,14 @@ instance FromNumber MouseButton Word8 where
   fromNumber Raw.SDL_BUTTON_X2     = ButtonX2
   fromNumber buttonCode            = ButtonExtra $ fromIntegral buttonCode
 
+instance ToNumber MouseButton Word8 where
+  toNumber ButtonLeft      = 0
```

Comment:
Can we use the enumerations from `Raw` instead of numeric constants here, like in the `FromNumber` instance above?